Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.m

```diff
@@ -96,6 +99,21 @@ + (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObject
     }
 }
 
++ (void)vok_addWithArrayInBackground:(NSArray *)inputArray completion:(VOKManagedObjectReturnBlock)completion
+{
+    [VOKCoreDataManager importArrayInBackground:inputArray
+                                       forClass:[self class]
+                                     completion:^(NSArray *arrayOfManagedObjectIDs) {
+                                         NSMutableArray *returnArray = [NSMutableArray array];
+                                         NSManagedObjectContext *moc = [[VOKCoreDataManager sharedInstance] managedObjectContext];
+                                         for (NSManagedObjectID *objectID in arrayOfManagedObjectIDs) {
+                                             [returnArray addObject:[moc objectWithID:objectID]];
+                                         }
+                                         completion([returnArray copy]);
```

Comment:
Interestingly, I'd take that article to be saying that the caller/value-receiver should copy, not the callee/value-sender.